go/tasks/plugins/webapi/agent/plugin_test.go

```diff
@@ -99,4 +100,106 @@ func TestPlugin(t *testing.T) {
 		ctx, _ = getFinalContext(context.TODO(), "CreateTask", &Agent{Endpoint: "localhost:8080", Timeouts: map[string]config.Duration{"CreateTask": {Duration: 1 * time.Millisecond}}})
 		assert.NotEqual(t, context.TODO(), ctx)
 	})
+
+	t.Run("test PENDING Status", func(t *testing.T) {
+		plugin := Plugin{
+			metricScope: fakeSetupContext.MetricsScope(),
+			cfg:         GetConfig(),
+		}
+		taskContext := new(webapiPlugin.StatusContext)
+
```

Comment:
Could we move this code to line 29, so we don't have to declare it again in each test?

Reply:
We have to declare "taskContext " in each test, but "plugin " is not needed, thanks a lot for review!